### ISSUE TITLE
Crash when calling `Instabug.logInfo("")`

### ISSUE THREAD

**Issue by alexkent** [Instabug/Instabug-iOS#139]
Occasionally I have seen this crasher. In this case it was in Xcode, with a unoptimised, debug build.

```
Thread 1Queue : com.apple.main-thread (serial)
#0	0x000000018efeef30 in objc_msgSend ()
#1	0x0000000190f9aad0 in -[NSObject(NSKeyValueObservingPrivate) _changeValueForKey:key:key:usingBlock:] ()
#2	0x0000000190fe4818 in -[NSOperationQueue setSuspended:] ()
#3	0x00000001007c77d8 in -[IBGLogLogger readsWritesQueue] ()
#4	0x00000001007c63b0 in -[IBGLogLogger addLog:level:] ()
#5	0x0000000100782da8 in -[IBGDataStore addIBGLogWithFormat:args:level:] ()
#6	0x00000001007d941c in __34+[Instabug logMessage:args:level:]_block_invoke ()
#7	0x000000010072b544 in -[IBGOperationQueue addOperationWithBlock:] ()
#8	0x00000001007d939c in +[Instabug logMessage:args:level:] ()
#9	0x00000001007d8e6c in +[Instabug logInfo:] ()
#10	0x0000000100083ff0 in Log(String, level : LogLevel) -> () at /Users/alke/dev/HID/GoID-iOS/GoID/Log.swift:12
#11	0x0000000100056d98 in BluetoothMonitor.centralManagerDidUpdateState(CBCentralManager) -> () at /Users/alke/dev/MyCompany/MyApp-iOS/MyApp/BluetoothStatusMonitor.swift:29
#12	0x000000010005704c in @objc BluetoothMonitor.centralManagerDidUpdateState(CBCentralManager) -> () ()
```

**Comment by yousefhamza** [comment]
Are you doing any KVOs on your `NSOperationQueue`s?

**Comment by alexkent** [comment]
Elsewhere in the app we have KVO controlled `Operation` subclasses. But in this particular context `BluetoothMonitor` is not wrapped in an `Operation`, it is an NSObject subclass held in a var on a view controller. The `NSOperationQueue` mentions in the in the trace above, are in your code I guess?

**Comment by HeshamMegid** [comment]
@alexkent this has been fixed in v7.1. Please check it out and let us know if you're still experiencing any issues.
